templates/ai/worker/do/openai/getTldrawAiChangesFromSimpleEvents.ts

```diff
@@ -82,7 +86,7 @@ function getTldrawAiChangesFromSimpleCreateOrUpdateEvent(
 					x: shape.x,
 					y: shape.y,
 					props: {
-						richText: toRichText(shape.text ?? ''),
+						richText: toRichTextIfNeeded(shape.text ?? ''),
```

Comment:
shape.text is rich text sometimes? 🤔

Reply:
yeah this stuff is a nightmare. these "simple" shape definitions sometimes use rich text (e.g. geo does, arrow doesnt), and we can't convert rich -> plain cus this runs on the server and we don't have access to our tiptap stuff there :|